app/controllers/homepage_controller.rb

```diff
@@ -1,7 +1,33 @@
 class HomepageController < ContentController
+  before_action :fetch_dump_sizes, only: :index
+
   def index
     @page.title = 'Služby. Otvorené dáta &amp; API. &middot; Ekosystém.Slovensko.Digital'.html_safe
     @page.og.title = 'Služby. Otvorené dáta &amp; API.'.html_safe
     @page.og.description = 'Lepšie digitálne služby štátu nie sú sci-fi.'
   end
+
+  private
+
+  def fetch_dump_sizes
+    uri = URI('https://s3.eu-central-1.amazonaws.com/ekosystem-slovensko-digital-dumps/')
+
+    @dump_sizes = Rails.cache.fetch('dump_sizes') do
+      DEFAULT_DUMP_SIZES.map do |schema_name, default_size|
+        size = Net::HTTP.start(uri.host) { |http| (Integer(http.request_head(URI.join(uri, "#{schema_name}.sql.gz"))['content-length']) / 1_000_000).ceil rescue default_size }
+        [schema_name, "(#{size} MB)"]
+      end.to_h
+    end
+  end
+
+  DEFAULT_DUMP_SIZES = {
```

Comment:
asi neriesme default size ale dajme tam `?MB`

Reply:
default som vyhodil. pozeram na to a rozmyslam ci tam nechceme radsej namiesto otaznika nevypisat nic, len nazov? (nebilo by do oci, ze tam nieco chyba :D )

pre porovnanie:
![image](https://user-images.githubusercontent.com/38187837/87485259-abbd4e80-c638-11ea-8a6f-0797a3078546.png)

![image](https://user-images.githubusercontent.com/38187837/87485239-9ba56f00-c638-11ea-97b0-369033e2e02f.png)

![image](https://user-images.githubusercontent.com/38187837/87485329-d6a7a280-c638-11ea-9890-913d54661400.png)

Reply:
nechal by som otaznik, lebo ta chyba by sa stat nemala a takto si to aspon skor niekto z nas vsimne